skiko/src/commonMain/kotlin/org/jetbrains/skia/Bitmap.kt

```diff
@@ -1083,57 +1083,75 @@ class Bitmap internal constructor(ptr: NativePointer) : Managed(ptr, _FinalizerH
 
 
 @ExternalSymbolName("org_jetbrains_skia_Bitmap__1nGetFinalizer")
+@ModuleImport("./skiko.mjs", "org_jetbrains_skia_Bitmap__1nGetFinalizer")
```

Comment:
I mean these exports in .mjs file - https://github.com/JetBrains/skiko/pull/831/files#diff-7a659f4a1851f4832de78b077e897c0746e6faa61e90177a74efb905eca31338R9

It would be difficult to maintain it manually, therefore we'll have a compiler plugin to automate it.